inference-engine/src/mkldnn_plugin/mkldnn_exec_network.cpp

```diff
@@ -12,11 +12,10 @@
 #include "mkldnn_itt.h"
 #include "nodes/mkldnn_memory_node.hpp"
 #include <threading/ie_executor_manager.hpp>
-#if ((IE_THREAD == IE_THREAD_TBB) || (IE_THREAD == IE_THREAD_TBB_AUTO))
+#if 0 && ((IE_THREAD == IE_THREAD_TBB) || (IE_THREAD == IE_THREAD_TBB_AUTO))
```

Comment:
would it be a bit more elegant to add some local "#define ENABLE_TBB_EXECUTOR"? rather than have 'if 0' in multiple places

Reply:
so if some team like TBB wilol want to eanble the executor for their tests it is unclera whihc 0s should be removed